### PR TITLE
[FIX] Web : In Tour if elemtnt is not on window then it scroll down page...

### PR DESCRIPTION
**Pull request by thiteshm155**
... to get popup and element visible
- Task: https://www.odoo.com/web#id=11194&view_type=form&model=project.task&action=333&active_id=131
- Pad: https://pad.odoo.com/p/r.6pZ0CSOTbl8z87uT

### CODE REVIEW

**Comment by antonylesuisse** [comment]
Why not a position fixed in css. Dont include a library.

**Comment by rim-odoo** [comment]
@Gorash Can I have your opinion on this PR?

File: addons/web/static/src/js/tour.js [hunk 1]
```diff
@@ -562,6 +563,16 @@ var Tour = {
         var $dropZone = $(".oe_drop_zone").first();
         var dropPosition = $dropZone.position();
         $dropZone.trigger($.Event("mouseup", { which: 1, pageX: dropPosition.left, pageY: dropPosition.top }));
+    },
+    scrollIntoView: function (element){
+        // if element is not on window (screen) then it scroll down to get popup visible
```
Comment (Gorash) [inline addons/web/static/src/js/tour.js]: Please, could you rename element in $element to indicate that it is a jQuery variable. (easier to debug)
Thanks

Comment (htr-odoo) [inline addons/web/static/src/js/tour.js]: Done : Rename the element to $element.

**Comment by Gorash** [comment]
@rim-odoo : you can merge.

**Comment by rim-odoo** [comment]
Manually merged at https://github.com/odoo/odoo/commit/51cdceba3ed8d8fd0b71838c111110727d8da3d4

**Pull request closed without merging**